src/main/java/com/exadel/discountwebapp/cloud/CloudController.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.discountwebapp.cloud;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController()
+@RequestMapping("/api/cloud")
+@RequiredArgsConstructor
+public class CloudController {
+
+    private final ImageCloudService imageCloudService;
+
+    @PostMapping("/upload/image")
+    public String uploadImage(@RequestParam("image") MultipartFile image) {
+        return imageCloudService.upload(image);
+    }
+
+    @DeleteMapping("/destroy/image")
+    public boolean destroyImage(@RequestBody String url) {
```

Comment:
the image should be deleted by its id, not by url
we can pass this id in request url